yarn.lock

```diff
@@ -2647,19 +2647,12 @@
   resolved "https://registry.yarnpkg.com/@types/range-parser/-/range-parser-1.2.4.tgz#cd667bcfdd025213aafb7ca5915a932590acdcdc"
```

Comment:
why are there changes in the lock file? Can you stay on the same lock as the main branch?

Reply:
That was an accident, let me remove it